lib/workspace.rb

```diff
@@ -0,0 +1,70 @@
+require 'httparty'
+require 'awesome_print'
+require 'pry'
+require 'dotenv'
+Dotenv.load
+
+require_relative 'slack_api_error'
+require_relative 'recipient'
+require_relative 'channel'
+require_relative 'user'
+
+class Workspace
+  attr_reader :users, :channels, :selected
+  
+  def initialize
+    @users = User.list
+    @channels = Channel.list 
+    @selected = nil
+  end
+  
+  def select_channel(input: nil)
```

Comment:
A keyword argument is probably overkill here.  Positional parameters can be optional too.

```suggestion
  def select_channel(input = nil)
```

Though I'm not sure it makes sense to select a channel without giving a name.